app/helpers/heatmap_helper.rb

```diff
@@ -187,7 +193,11 @@ def self.fetch_top_taxons(
       phage_clause = " AND is_phage = 1"
     end
 
+    # Select both species and genus level counts.
+    # Taxon counts are later filtered down to the desired taxon level in samples_taxons_details and top_taxons_details
+    # by calling only_species_or_genus_counts!, OR will be filtered on the client-side if a user has "heatmap_filter_fe" enabled.
```

Comment:
Dont forget to update the comments given the last change.

Reply:
Oops, good catch 😅